ports/ceferino/Don Ceferino Hazaña.sh

```diff
@@ -24,20 +24,17 @@ BINARY=ceferino
 
 > "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
 
-mkdir -p "$GAMEDIR/conf"
```

Comment:
Why?

Reply:
This directory is already part of the port's directory and therefore will be should be created when installed. There is also a configuration file within:
https://github.com/PortsMaster/PortMaster-New/blob/main/ports/ceferino/ceferino/conf/.ceferino

I did notice a `.gitkeep` in same directory so maybe this `.ceferino` file wasn't initially planned to be there?